packages/ia-components/sandbox/audio-players/utils/youtube-full-album-registry.js

```diff
@@ -0,0 +1,35 @@
+import youTubeParamsParser from './youtube-params-parser';
+/**
+ * YouTube Full Album Registry
+ * checks playlist and if:
+ * - all IDs are the same
+ * - has timestamp
+ * it will create a dictonary { trackNumber: {videoID, timestamp} }
+ *
+ *
+ *
+ *
+ */
+export default (playlist) => {
+  let mainVideoID = '';
+  const directory = playlist.reduce((acc, track, index) => {
+    const dir = acc || {};
```

Comment:
**Suggestion**: Since you're passing in a starting object literal, you don't need this check. You can rename `acc` to `dir`, drop this line, and continue as-is.

Reply:
this failed at lint, `es-no-param-reassign` - I'll update to the above and when we do our linter rules scrub, we will have a note to remember.